dev/io.openliberty.microprofile.health.3.1.internal_fat/fat/src/io/openliberty/microprofile/health31/fat/SlowAppStartupHealthCheckTest.java

```diff
@@ -218,6 +218,11 @@ public void run() {
 
         }
 
+        log("testStartupEndpointOnServerStart", "Waiting for Application to start message, after Health check reports 200.");
+        String line = server1.waitForStringInLog("(CWWKZ0001I: Application DelayedHealthCheckApp started)+", 60000);
+        log("testSlowAppStartUpHealthCheck", "Application Started message found: " + line);
```

Comment:
Should the `assertNotNull()` goes before the log line?  Otherwise, you will get "Application Started message found: null"

Reply:
Right, I updated the code.